### DISCUSSION TITLE
Item blacklist

### DISCUSSION THREAD

**Discussion by ghost** [FakeFishGames/Barotrauma#8730] [Ideas and suggestions]
Would be nice to see the ability to blacklist certain items from naturally spawning or being crafted through a server's settings. I host an RP server where we have to ban a few modded items from being used because they're overpowered and we can't just remove the mod, because we use other items from it.

**Comment by esirprus** [comment]
I think this is very important and needs to be on the agenda
1. add some challenging gameplay by disabling some items
2. it would prevent some people from messing up, such as making radioactive fuel rods and sneaking them away to cause radiation sickness in others
3. as WrillWasTaken said, some mods have individual items that are overpowered, while others are useful and balanced (like the EK series of mods)
4. Sometimes it is difficult to contact the author to request changes to the strength of an item.
5. Sometimes different items from multiple mods can interact powerfully and it is not possible to expect compatibility between mod authors
